ArduCopter/mode_flowhold.cpp

```diff
@@ -369,7 +369,8 @@ void ModeFlowHold::update_height_estimate(void)
 
     // get delta velocity in body frame
     Vector3f delta_vel;
-    if (!copter.ins.get_delta_velocity(delta_vel)) {
+    float delta_vel_dt;  // FIXME: this should be used!
```

Comment:
why should it be used? It wants the change in velocity. It doesn't want the rate of change, so the time doesn't matter